group.go

```diff
@@ -53,11 +53,18 @@ type Scalar interface {
 	// Set to a fresh random or pseudo-random scalar
 	Pick(rand cipher.Stream) Scalar
 
-	// SetBytes will take bytes and create a scalar out of it
+	// SetBytes sets the scalar from a big-endian byte-slice,
+	// reducing if necessary to the appropriate modulus.
 	SetBytes([]byte) Scalar
 
-	// Bytes returns the raw internal representation
+	// Bytes returns a big-Endian representation of the scalar
 	Bytes() []byte
+
+	// Allow or disallow use of faster variable-time implementations
+	// of operations on this Point, returning the old flag value.
+	// This flag always defaults to false (constant-time only)
+	// in implementations that can provide constant-time operations.
+	SetVarTime(varTime bool) bool
```

Comment:
Perhaps it would be worth taking a bit of a "census" throughout the crypto and/or onet code to see where all we do Scalar arithmetic and how much of that arithmetic might conceivably benefit from being able to use a faster variable-time version for non-sensitive calculations?  If it turns out there's not all that much, and/or it proves hard to identify and disentangle which are the "sensitive" vs "non-sensitive" scalars, then I would propose in this case it's probably best (certainly safest) just to make the scalars always use constant-time arithmetic and take whatever performance hit that means.

Given that point.Mul is the main performance-critical operation we're concerned about in general, perhaps keeping that as the only one we bother with having constant-time and variable-time versions of would be perfectly fine.